lib/routes/qztc/jwc/index.ts

```diff
@@ -0,0 +1,121 @@
+import { Data, Route } from '@/types';
+import cache from '@/utils/cache';
+import { parseDate } from '@/utils/parse-date';
+import { load } from 'cheerio';
+import ofetch from '@/utils/ofetch';
+import timezone from '@/utils/timezone';
+
+const rootUrl = 'https://www.qztc.edu.cn/jwc/';
+const host = 'www.qztc.edu.cn';
+const name = '泉州师范学院-教务处';
+
+export const route: Route = {
+    path: '/jwc/:type',
+    categories: ['university'],
+    example: '/qztc/jwc/1939',
+    parameters: { type: '分类，见下表' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: String(name),
+    maintainers: ['iQNRen'],
+    url: 'www.qztc.edu.cn',
+    handler,
+    radar: [
+        {
+            source: ['www.qztc.edu.cn/:type/list.htm'],
```

Comment:
This doesn't match the url in https://github.com/DIYgod/RSSHub/blob/63ca61a4450276d53a6e62fe5b233ba26cbc666c/lib/routes/qztc/jwc/index.ts#L8